### PR TITLE
Rename getNodeFromMedia and getFaceFilter to use the same naming conventions

### PR DESCRIPTION
**Pull request by AndreiEvstratenko**
# Pull Request

## Description

- When using the integrating a 3rd party API, I want to have it work as simple and predictable as possible, so I don't have to worry about weird intricacies of integrating, and instead build great features.

### Motivation and Context

- closes #2225

### Types of changes

- [ ] Bug fix
- [ ] New feature
- [x] Maintenance

### Checklist

- [x] Documentation
- [ ] Tests

### CODE REVIEW

File: SvrfSDK/Source/SvrfSDK.swift [hunk 1]
```diff
@@ -305,43 +305,43 @@ public class SvrfSDK: NSObject {
...
-        - media: The *Media* to generate the face filter from. The *type* must be `_3d`.
+        - media: The *Media* to generate the node from. The *type* must be `_3d`.
         - success: Success closure.
         - faceFilter: The *SCNNode* that contains face filter content.
         - failure: Error closure.
         - error: A *SvrfError*.
      */
-    public static func getFaceFilter(with media: Media,
-                                     onSuccess success: @escaping (_ faceFilter: SCNNode) -> Void,
+    public static func getNode(with media: Media,
```
Comment (brentchow) [inline SvrfSDK/Source/SvrfSDK.swift]: I think this would be better named `generateFaceFilterNode`

Comment (AndreiEvstratenko) [reply SvrfSDK/Source/SvrfSDK.swift]: changed

File: SvrfSDK/Source/SvrfSDK.swift [hunk 2]
```diff
@@ -254,17 +254,17 @@ public class SvrfSDK: NSObject {
      Generates a *SCNNode* for a *Media* with a *type* `_3d`. This method can used to generate
      the whole 3D model, but is not recommended for face filters.
      
-     - Attention: Face filters should be retrieved using the `getFaceFilter` method.
+     - Attention: Face filters should be retrieved using the `getNode` method.
```
Comment (brentchow) [inline SvrfSDK/Source/SvrfSDK.swift]:
```suggestion
     - Attention: Face filters should be retrieved using the `getFaceFilterNode` method.
```

File: SvrfSDK/Source/SvrfSDK.swift [hunk 3]
```diff
@@ -305,43 +305,43 @@ public class SvrfSDK: NSObject {
...
-        - media: The *Media* to generate the face filter from. The *type* must be `_3d`.
+        - media: The *Media* to generate the node from. The *type* must be `_3d`.
         - success: Success closure.
         - faceFilter: The *SCNNode* that contains face filter content.
         - failure: Error closure.
         - error: A *SvrfError*.
      */
-    public static func getFaceFilter(with media: Media,
-                                     onSuccess success: @escaping (_ faceFilter: SCNNode) -> Void,
+    public static func generateFaceFilterNode(with media: Media,
```
Comment (brentchow) [inline SvrfSDK/Source/SvrfSDK.swift]: should `with` be changed to `for`? you used that name above.

File: SvrfSDK/Source/SvrfSDK.swift [hunk 4]
```diff
@@ -305,43 +305,43 @@ public class SvrfSDK: NSObject {
...
+                                     onSuccess success: @escaping (_ node: SCNNode) -> Void,
```
Comment (brentchow) [inline SvrfSDK/Source/SvrfSDK.swift]: Do you think `node` is less confusing than `faceFilter`?

File: SvrfSDK/Source/SvrfSDK.swift [hunk 5]
```diff
@@ -305,43 +305,43 @@ public class SvrfSDK: NSObject {
...
                     }
 
-                    faceFilter.morpher?.calculationMode = SCNMorpherCalculationMode.normalized
+                    node.morpher?.calculationMode = SCNMorpherCalculationMode.normalized
 
-                    success(faceFilter)
+                    success(node)
 
-                    SEGAnalytics.shared().track("Face Filter Node Requested",
+                    SEGAnalytics.shared().track("Node Requested",
```
Comment (brentchow) [inline SvrfSDK/Source/SvrfSDK.swift]: Do not change this.

Comment (AndreiEvstratenko) [reply SvrfSDK/Source/SvrfSDK.swift]: done

File: SvrfSDK/Source/SvrfSDK.swift (hunk 2 above)
Comment (AndreiEvstratenko) [reply SvrfSDK/Source/SvrfSDK.swift]: done

File: SvrfSDK/Source/SvrfSDK.swift (hunk 4 above)
Comment (AndreiEvstratenko) [reply SvrfSDK/Source/SvrfSDK.swift]: "faceFilter" more convenient but it's a good practice to include a type (`SCNNode` in our case into variable's name). Let's name this "faceFilterNode".

File: SvrfSDK/Source/SvrfSDK.swift (hunk 3 above)
Comment (AndreiEvstratenko) [reply SvrfSDK/Source/SvrfSDK.swift]: changed

File: SvrfSDK/Source/SvrfSDK.swift [hunk 6]
```diff
@@ -305,41 +305,41 @@ public class SvrfSDK: NSObject {
...
-    
+
     /**
      The SVRF API allows you to access all of SVRF's ARKit compatible face filters and stream them directly to your app.
-     Use the `getFaceFilter` method to stream a face filter to your app and convert it into a *SCNNode* in runtime.
+     Use the `generateFaceFilterNode` method to stream a face filter to your app and convert it into a *SCNNode* in runtime.
      
      - Parameters:
-        - media: The *Media* to generate the face filter from. The *type* must be `_3d`.
+        - media: The *Media* to generate the node from. The *type* must be `_3d`.
```
Comment (brentchow) [inline SvrfSDK/Source/SvrfSDK.swift]:
```suggestion
        - media: The *Media* to generate the face filter node from. The *type* must be `_3d`.
```

File: README.md [hunk 7]
```diff
@@ -243,7 +243,7 @@ Get a face filter *SCNNode* for *Media* with ID "547963".
 
 ```swift
 SvrfSDK.getMedia(id: "547963", onSuccess: { media in
-    SvrfSDK.getFaceFilter(with: media, onSuccess: { faceFilter in
+    SvrfSDK.generateFaceFilterNode(with: media, onSuccess: { faceFilter in
```
Comment (brentchow) [inline README.md]:
```suggestion
    SvrfSDK.generateFaceFilterNode(for: media, onSuccess: { faceFilterNode in
```

File: SvrfSDK/Source/SvrfSDK.swift [hunk 8]
```diff
@@ -305,41 +305,41 @@ public class SvrfSDK: NSObject {
...
+                                     onSuccess success: @escaping (_ faceFilterNode: SCNNode) -> Void,
                                      onFailure failure: Optional<(_ error: SvrfError) -> Void> = nil) {
 
             if media.type == ._3d, let glbUrlString = media.files?.glb, let glbUrl = URL(string: glbUrlString) {
                 let modelSource = GLTFSceneSource(url: glbUrl)
 
                 do {
-                    let faceFilter = SCNNode()
-                    let node = try modelSource.scene().rootNode
+                    let node = SCNNode()
```
Comment (brentchow) [inline SvrfSDK/Source/SvrfSDK.swift]: shouldn't this be `faceFilterNode` now?

File: SvrfSDK/Source/SvrfSDK.swift (hunk 6 above)
Comment (AndreiEvstratenko) [reply SvrfSDK/Source/SvrfSDK.swift]: done

File: SvrfSDK/Source/SvrfSDK.swift (hunk 8 above)
Comment (AndreiEvstratenko) [reply SvrfSDK/Source/SvrfSDK.swift]: makes sense. changed

**Pull request merged**